docker-machine-nfs.sh

```diff
@@ -146,15 +146,15 @@ configureNFS()
     echoSuccess "\t\t\t\t\t\tOK"
 }
 
-# @info:    Configures the Boot2Docker to mount nfs
-configureBoot2Docker()
+# @info:    Configures the Docker Machine to mount nfs
+configureDockerMachine()
```

Comment:
Would this be better renamed to `configureVirtualboxDriver`? That would open the door for other docker machines to be connected via nfs.